src/components/Data/data.js

```diff
@@ -18,7 +19,7 @@ const StyledButton = styled(Button)`
 
 export const data = [
   {
-    cta: <StyledButton label="Add points" />,
+    cta: <Link to="/hpe-softcat-leaderboard/pointsform"><StyledButton label="Add points" /></Link>,
```

Comment:
Could we change this to submit sales as well
```suggestion
    cta: <Link to="/hpe-softcat-leaderboard/pointsform"><StyledButton label="Submit Sales" /></Link>,
```